packages/discord-bot/src/utils/assertPraiseGiver.ts

```diff
@@ -0,0 +1,53 @@
+import { CacheType, CommandInteraction, GuildMember } from 'discord.js';
+import { dmError, praiseRoleError } from '../utils/praiseEmbeds';
+import { settingValue } from 'api/dist/shared/settings';
+
+export const assertPraiseGiver = async (
+  praiseGiver: GuildMember,
+  interaction: CommandInteraction<CacheType>,
+  sendReply: boolean
+): Promise<boolean> => {
+  const praiseGiverRoleIDRequired = (await settingValue(
+    'PRAISE_GIVER_ROLE_ID_REQUIRED'
+  )) as boolean;
+  const praiseGiverRoleID = (await settingValue(
+    'PRAISE_GIVER_ROLE_ID'
+  )) as string;
+
+  if (praiseGiverRoleIDRequired) {
```

Comment:
Could we make this first conditional return a value, instead of wrapping all the others? Just a bit easier to read.

i.e.  
```
  if (!praiseGiverRoleIDRequired) {
   return true;
  }
  
  ...
  ```

Reply:
👍 done